modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/web/DetectorController.java

```diff
@@ -65,6 +69,7 @@ public DetectorDocument findByUuid(@RequestParam String uuid) {
     public List<DetectorDocument> findByCreatedBy(@RequestParam String user) {
         List<DetectorDocument> detectors = detectorRepo.findByCreatedBy(user);
         if (detectors == null || detectors.isEmpty()) {
+            exceptionCounter.increment(1.0);
```

Comment:
@tusharbahl This doesn't track the exceptions which are thrown in detector repo and detector mapping impl classes.